src/main/java/org/broadinstitute/consent/http/db/InstitutionDAO.java

```diff
@@ -46,6 +46,12 @@ void updateInstitutionById(@Bind("institutionId") Integer institutionId,
   @SqlQuery("SELECT * FROM institution WHERE institution_id = :institutionId")
   Institution findInstitutionById(@Bind("institutionId") Integer institutionId);
 
-  @SqlQuery("SELECT * FROM institution")
+  @SqlQuery("SELECT i.*, u.*, u2.dacuserid AS updateUserId, u2.email AS updateUserEmail, " +
+  " u2.displayName AS updateUserName, u2.createdate AS updateUserCreateDate, " +
+  " u2.additional_email as updateUserAdditionalEmail, u2.email_preference as updateUserEmailPreference, " +
+  " u2.status as updateUserStatus, u2.rationale as updateUserRationale " +
+  " FROM institution i" +
+  " LEFT JOIN dacuser u ON u.dacuserid = i.create_user" +
+  " LEFT JOIN dacuser u2 ON u2.dacuserid = i.update_user")
   List<Institution> findAllInstitutions();
```

Comment:
aliased the update user's properties so the mapper could understand

Reply:
Using the institution mapper is probably the more complicated approach. When we have joins, we should try to use RowReducers instead since they manage a lot of the conditional logic for you.

Reply:
Look at how we're mapping user roles and users in `UserDAO` for some guidance. One thing we could be doing better in that code is to use column prefixes to disambiguate the joins - see https://jdbi.org/#_registerbeanmapper for docs on that.

Reply:
Okay, I'm not exactly sure how to do this, but I'll commit my best attempt and maybe we can talk about it tomorrow

Reply:
I'd be happy to help too. Give it a shot and we can work together tomorrow on it.

Reply:
I based this class off of UserWithRolesReducer, when I test it out on swagger this is the error I get: "Mapping bean class org.broadinstitute.consent.http.models.User didn't find any matching columns in result set"
I also just commented out all the work I did in the Institution mapping class instead of deleting it because I wasn't sure if we would need any of it in implementing the reducer